### DISCUSSION TITLE
GGP signed but the commits are shown Unverified

### DISCUSSION THREAD

**Discussion by Kritikarao77** [community/community#39911] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a valid gpg set for my account, and I use that to sign my commits.
However once pushed to the repo, they are seen as unverified there.

anyone faced this or can help resolve this pls?

Thank you.

**Comment by MitkoVtori** [comment]
If you push your commits through your computer files, to the repo It mait have problems with the verification. If so, you can just push your changes in a branch and then make a pull request. If that is not the problem, check [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)
Hope that answers your question!

**Comment by airtower-luna** [comment, marked as answer]
If you click the "unverified" label you should get some more information on *why* it can't be verified. You don't have any public repositories, so it's impossible to check from the outside. :sweat_smile: 

You can also check `git log --show-signature` locally to check if the signatures and signing keys are as you expect.
